src/smokeHandlers.ts

```diff
@@ -271,7 +260,8 @@ export function SetupMainHandlers()
         const target = event.target as HTMLInputElement;
         if (target.value === "fast")
         {
-            await OBR.notification.show("Notice: Issues can occur when using FAST Performance with PERSISTENCE, specifically with revealing inner fogged areas when traveling in a full circle around them. Use both with caution and care.", "DEFAULT");
+            // This has been fixed, so lets not scare anyone away anymore:
```

Comment:
I'm cool with flat-out deleting things that are no longer relevant.  I don't particularly like having a lot of left-over code from bygone times.

Reply:
Same, just wanted to have some confidence that fast mode is 100% now before changing the default, though don't think this needs to stay